charts/tidb-cluster/templates/config/_privileged-tidb-config.tpl

```diff
@@ -228,10 +231,15 @@ grpc-keepalive-timeout = 3
 # max time for commit command, must be twice bigger than raft election timeout.
 commit-timeout = "41s"
 
-[binlog]
+[txn-local-latches]
+# Enable local latches for transactions. Enable it when
+# there are lots of conflicts between transactions.
+enabled = false
+capacity = 10240000
 
-# Socket file to write binlog.
-binlog-socket = ""
+[binlog]
+# enable to write binlog.
+enable = false
```

Comment:
should this field be managed in Values.yaml

Reply:
The command line arguments override the configuration file.